packages/react-examples/src/react-menu/MenuList/MenuList.stories.tsx

```diff
@@ -1,9 +1,7 @@
 import * as React from 'react';
 
-import { MenuList, MenuItem } from '@fluentui/react-menu';
-import { teamsLightTheme } from '@fluentui/react-theme';
-import { FluentProvider } from '@fluentui/react-provider';
-import { CutIcon, PasteIcon, EditIcon } from '@fluentui/react-icons-mdl2';
+import { MenuList, MenuItem, MenuItemCheckbox } from '@fluentui/react-menu';
+import { CutIcon, PasteIcon, EditIcon, AcceptIcon } from '@fluentui/react-icons-mdl2';
```

Comment:
Aren't these icons old style? We need to be using Fluent icons. I can provide if needed.

Reply:
I don't exactly know what the story is for icons using convergence right now, and these are used in the stories not in the component itself, so it's still pretty flexible when that story is defined

Reply:
I see MDL2 icons are used elsewhere as well so I leave it at that